hazelcast/include/hazelcast/client/sql/sql_result.h

```diff
@@ -153,6 +231,10 @@ class HAZELCAST_API sql_result : public std::enable_shared_from_this<sql_result>
      */
     page_iterator iterator();
 
+    page_iterator_sync pbegin(
```

Comment:
It might throw exceptions like illegal_state, could you document it?

Reply:
Fixed.